[a11y] fix page heading structure in management apps

alisonelizabeth opened the pull request:
This PR fixes the page heading structure across apps owned by the Elasticsearch UI team.

elasticmachine commented:
Pinging @elastic/es-ui (Team:Elasticsearch UI)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/6135/)
* Commit: 4be876208d1f1dbeec8d1a243e6ea1b34eaa56b9

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/6143/)
* Commit: 04f1f908597313001c717c89c3e65e7f0e4b8187

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/6185/)
* Commit: c1ac31f1d8495e94a817aa177064bfedb0d0b2db

myasonik commented:
@alisonelizabeth Do you have a list of URLs I can check this on? That'd help me proof this much quicker. 🙏

alisonelizabeth commented:
@myasonik sure thing!

Anything that has a wizard, you will need to proceed through it to test all the steps.

**Console**
-  /app/kibana#/dev_tools/console?_g=()

**Index Lifecycle Policies**
- /app/kibana#/management/elasticsearch/index_lifecycle_management/policies/edit/<ilm_policy_name>


**Index Management**
- /app/kibana#/management/elasticsearch/index_management/create_template

**License Management**
- /app/kibana#/management/elasticsearch/license_management/home?_g=()

**Rollup Jobs**
- /app/kibana#/management/elasticsearch/rollup_jobs/create

**Snapshot & Restore**
- /app/kibana#/management/elasticsearch/snapshot_restore/add_repository
- /app/kibana#/management/elasticsearch/snapshot_restore/add_policy

**Watcher**
- /app/kibana#/management/elasticsearch/watcher/watches/new-watch/threshold
- /app/kibana#/management/elasticsearch/watcher/watches/new-watch/json (Navigate to the `Simulate` tab to test changes there)

alisonelizabeth commented:
@myasonik thanks for the review! I addressed your feedback. Mind taking another quick look? Changes made in commit https://github.com/elastic/kibana/pull/48936/commits/b698a258629775c5879c2c6a1cf8dbb2595a2c33.

alisonelizabeth commented:
@elasticmachine merge upstream

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/6720/)
* Commit: b698a258629775c5879c2c6a1cf8dbb2595a2c33

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/6731/)
* Commit: dbe3ba40433db74c87e61b7ddc0697618ee85791
